src/main/java/com/minecolonies/coremod/colony/crafting/LootTableAnalyzer.java

```diff
@@ -166,6 +170,20 @@ private static List<LootDrop> entryToDrops(@Nullable final LootTables lootTableM
         return drops;
     }
 
+    private static boolean conditionsSeemImpossible(@NotNull final JsonArray conditions)
```

Comment:
missing javadoc

Reply:
A lot of private methods lack javadoc.  But k.

Reply:
good call, if you want you could add those aswell

Reply:
I meant in general in the entire mod 😛

Reply:
hey I wouldnt complain if you added those :)